releaseNotes/AzureAppConfigurationProviderPython.md

```diff
@@ -2,6 +2,12 @@
 
 [Source code][source_code] | [Package (Pypi)][package] | [Samples][samples]
 
+## 2.0.0b3 - November 14, 2024
+
+### Bug Fixes
+
+* Changed the method in which allocation id is generated for feature flags.
```

Comment:
Updated the method for generating allocation IDs for feature flag telemetry to ensure consistency across different languages of App Configuration providers.